__tests__/routes/user.test.js

```diff
@@ -51,4 +51,73 @@ describe('users', () => {
     expect(res.status).toBe(401);
     expect(res.body.message).toBe('test@email.com has already been confirmed');
   });
+
+  // Forget password tests
+  test('Password reset link sent sucessfully', async () => {
+    const res = await request(app)
+      .post(`${urlPrefix}/users/forget`)
+      .send({ user: { email: 'test@email.com' } });
+    expect(res.status).toBe(201);
+    expect(res.body.message).toBe('Password reset link sent sucessfully. Please check your email!');
+  }, 30000);
+
+  test('No user found with that email address', async () => {
+    const res = await request(app)
+      .post(`${urlPrefix}/users/forget`)
+      .send({ user: { email: 'fake@email.com' } });
+    expect(res.status).toBe(404);
+    expect(res.body.message).toBe('No user found with that email address');
+  });
+  test('Bad request- password forget', async () => {
```

Comment:
Empty line after line 70. Why have you spaced the rest of the tests but not this one?

Reply:
It was unintentionally! let me change that too